cmd/cert-checker/main_test.go

```diff
@@ -118,13 +144,16 @@ func TestCheckCert(t *testing.T) {
 }
 
 func TestGetAndProcessCerts(t *testing.T) {
-	dbMap, err := sa.NewDbMap(dbConnStr)
+	saDbMap, err := sa.NewDbMap(saDbConnStr)
 	test.AssertNotError(t, err, "Couldn't connect to database")
-	checker := newChecker(dbMap)
-	sa, err := sa.NewSQLStorageAuthority(dbMap)
+	paDbMap, err := sa.NewDbMap(paDbConnStr)
+	test.AssertNotError(t, err, "Couldn't connect to policy database")
+	checker := newChecker(saDbMap, paDbMap, false)
+	sa, err := sa.NewSQLStorageAuthority(saDbMap)
 	test.AssertNotError(t, err, "Couldn't create SA to insert certificates")
 	defer func() {
-		dbMap.TruncateTables()
+		saDbMap.TruncateTables()
+		paDbMap.TruncateTables()
```

Comment:
In another PR that comes after this one (and is not blocking it), please refactor all of these setups into a single function.